### ISSUE TITLE
Slicing Prints

### ISSUE THREAD

**Issue by bobbypop999** [donovan6000/M33-Fio#200]
When you have got to the point when you choose the quality and infill the manual settings it only shows infill, layerheight. I am using the currant version of octoprint and M33 Fio, safari mobile.
If I use Crome it has more options like the version before.

**Comment by donovan6000** [comment]
Can you post a screenshot? I don't have access to anything that can run safari mobile, so I might not be able to fix this myself unless it's something simple like adding vendor prefix names to some of the CSS properties.

**Comment by dieideeistgut** [comment]
I am running into several issues with Safari and also Firefox too - current Chrome behaves nice though. I'm actually thinking that these issues come from within octoprint and not from M33-Fio.

**Comment by donovan6000** [comment]
Can you guys test out M33 Fio's devel version and let me know if any of the visual issue were resolved?

You can install it by install a plugin from this url `https://github.com/donovan6000/M33-Fio/archive/devel.zip` using OctoPrint's builtin plugin manager.

**Comment by bobbypop999** [comment]
Yes I will have a go but it will be a wekk before I get back to you.

**Comment by bobbypop999** [comment]
Unfortuanlly it doesn't work but here is a screen shot for more info.
![image](https://cloud.githubusercontent.com/assets/20486686/20844215/b53ebd4c-b8b6-11e6-8608-13741b2e5b1a.png)

**Comment by bobbypop999** [comment]
Found another problem! After the first page of slicing you can't even get to the quality settings it just starts slicing?

**Comment by donovan6000** [comment]
I'm not going to be able to fix this myself since I don't have access to anything running an older version of Safari mobile. M33 Fio just wasn't designed for outdated web browsers, and the version of Safari mobile that your using doesn't support some of the modern CSS features that M33 Fio uses.

If anyone's interested in trying to fix this issue, then [here's M33 Fio's css file](https://github.com/donovan6000/M33-Fio/blob/master/octoprint_m33fio/static/css/m33fio.css).

**Comment by bobbypop999** [comment]
Ok my iPad 2 is outdated so if I could upgrade to ios10 maybe that could solve issues?

**Comment by donovan6000** [comment]
If ios10 comes with a newer version of Safari mobile then it could fix the issue.

**Comment by bobbypop999** [comment]
Just realised I have access to iOS 10 on another doe use I will try it immediately.

**Comment by bobbypop999** [comment]
Tried but didn't work!!! Same screen as last time argh!!!

**Comment by bobbypop999** [comment]
I have just tried the slicer plug in works very good also it can change the print speed and more.
Until this might get fixed I think I will use the plug in instead.

**Comment by bobbypop999** [comment]
Now I only get this
![image](https://cloud.githubusercontent.com/assets/20486686/20861229/726c7d5c-b982-11e6-966b-3eaa1af57b07.png)

**Comment by donovan6000** [comment]
@bobbypop999 Update to the latest version [V1.13](https://github.com/donovan6000/M33-Fio/releases/tag/V1.13).

**Comment by bobbypop999** [comment]
How I can't get to the settings?

**Comment by donovan6000** [comment]
Your web browser is detecting a JavaScript error when parsing M33 Fio's JavaScript, and it halts JavaScript execution on the page when that happens. So you'll need to manually uninstall M33 Fio since you wont be able to do it from the web interface due to the JavaScript error.

If your using OctoPi, then you can SSH into the Raspberry Pi and run the following command to uninstall M33 FIo and restart OctoPrint. Then you'll be able to use the web interface again to install the latest version of M33 Fio.
```
/home/pi/oprint/bin/pip uninstall OctoPrint-M33Fio
service octoprint start
```

**Comment by bobbypop999** [comment]
How do I ssh in Windows?

**Comment by donovan6000** [comment]
You can use [Putty](https://www.raspberrypi.org/documentation/remote-access/ssh/windows.md).

**Comment by bobbypop999** [comment]
I was just woundering if you were to have the heated bed that chase has made pluged into the raspberry pi after you slice a model and click print normaly a window that asks for nozzle temp and wave bonding comes up but with the heated bed does it also have bed temp there? Also could you add vase mode to the slicing?

**Comment by sam-dyer** [comment]
Just to let you know on ios 11 every thing works fine!

![8984d416-95cd-4458-a258-702204aa5978](https://user-images.githubusercontent.com/20486686/30783724-c5ed891c-a13f-11e7-8bf9-030a6ff0b825.jpeg)
